ci-operator/config/terraform-redhat/terraform-provider-ocm/terraform-redhat-terraform-provider-ocm-main.yaml

```diff
@@ -27,6 +36,23 @@ tests:
     make unit-test
   container:
     from: src
+- as: e2e-presubmit
+  steps:
+    cluster_profile: aws-qe
+    env:
+      GATEWAY_URL: https://api.stage.openshift.com
+      OPENSHIFT_VERSION: "4.13"
+      TOKEN_URL: https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Comment:
you can delete it. it's the same default values you defined in the step

Reply:
can you set those values as the defaults for ``ci-operator/step-registry/rosa/terraform/aws/e2e/test/rosa-terraform-aws-e2e-test-ref.yaml``?
at least for ``GATEWAY_URL`` and ``TOKEN_URL``, I don't see us working with different envs in the near future